### ISSUE TITLE
Uncaught Exception: You have requested a non-existent parameter "api_private_key"

### ISSUE THREAD

**Issue by muyncky** [blauwfruit/juice#33]
```log
2024-08-16 16:56:38 juice-apache-1  | [Fri Aug 16 14:56:38.804316 2024] [php7:notice] [pid 22] [client 192.168.65.1:25521] 2024-08-16T16:56:38+02:00 [critical] Uncaught Exception: You have requested a non-existent parameter "api_private_key".
```

Possible solution before upgrade adding the following details:

```php
<?php return array (
...
    'api_private_key' => '',
    'api_public_key' => '',
  ),
);
```

**Comment by muyncky** [comment]
Might be fixed by #32

**Comment by muyncky** [comment]
Fixed by #34
